<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneWindow</class>
 <widget class="QMainWindow" name="PhoneWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhoneWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabContact">
     <attribute name="title">
      <string>Contacts</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <widget class="QLabel" name="labelRecord">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tabInfoAboutCall">
     <attribute name="title">
      <string>Active Calls</string>
     </attribute>
     <widget class="QLabel" name="labelMembers">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Members</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTime">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>131</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelGroup">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCall">
       <property name="text">
        <string>Call</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonHangUp">
       <property name="text">
        <string>Hang Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonConference">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Conference</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonHold">
       <property name="text">
        <string>Hold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRetreive">
       <property name="text">
        <string>Retrieve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
